app/api/posts/route.ts

```diff
@@ -5,7 +5,7 @@ import { prismaPost } from '@/lib/createPost'
 
 export const POST = auth(async (req) => {
     const data = await req.json()
-    if (req.auth) {
+    if (req.auth || process.env.NEXT_PUBLIC_USE_TEST === 'true') {
```

Comment:
Is it possible to mock `req.auth` so you don't need to use the flag here?

Reply:
Put down what you found in your investigation here

Reply:
After attempting to mock `req.auth`, I realized that cypress only lets you mock the response object. As of right now, I am unable to mock the `req.auth` and the flag will be needed.